engine/ecs/private/ecs_module.cpp

```diff
@@ -0,0 +1,55 @@
+#include "ecs_module.hpp"
+#include "components/relationship_helpers.hpp"
+// #include "time_module.hpp"
+
+ModuleTickOrder ECSModule::Init(MAYBE_UNUSED Engine& engine)
+{
+    TransformHelpers::SubscribeToEvents(registry);
+    RelationshipHelpers::SubscribeToEvents(registry);
+    return ModuleTickOrder::eTick;
+}
+
+void ECSModule::Shutdown(MAYBE_UNUSED Engine& engine)
+{
+    TransformHelpers::UnsubscribeToEvents(registry);
+    RelationshipHelpers::UnsubscribeToEvents(registry);
+}
+
+void ECSModule::Tick(Engine& engine)
+{
+    // auto dt = engine.GetModule<TimeModule>().GetDeltatime().count();
+    float dt = 16.66f;
```

Comment:
Forgot to uncomment getting deltaTime from TimeModule?

Reply:
Yeah did, good catch!